crates/consensus/beacon/src/engine/event.rs

```diff
@@ -9,7 +9,7 @@ pub enum BeaconConsensusEngineEvent {
     /// The fork choice state was updated.
     ForkchoiceUpdated(ForkchoiceState, ForkchoiceStatus),
     /// A block was added to the canonical chain.
-    CanonicalBlockAdded(Arc<SealedBlock>),
+    CanonicalBlockAdded(Arc<SealedBlock>, Duration),
```

Comment:
I'm okay with this but this should be documented what the duration is

Reply:
Documented